docs/docs/use-static-query.md

```diff
@@ -45,7 +45,60 @@ export default () => {
 }
 ```
 
+### Advanced Example
+
+The potential of `useStaticQuery` is to make easier reuse queries across your code.
+
+One of the most used queries is to retrieve your site metadata.
+
+You can create a `useSiteMetadata` for that purpose:
+
+```jsx:title=src/components/hook/site-meta.js
+import { useStaticQuery, graphql } from "gatsby"
+
+export const useSiteMetadata = () => {
+  const { site } = useStaticQuery(
+    graphql`
+      query SiteMetaData {
+        site {
+          siteMetadata {
+            siteUrl
+            headline
+            description
+            image
+            video
+            twitter
+            name
+            logo
+          }
+        }
+      }
+    `
+  )
+  return site.siteMetadata
+}
+```
+
+To use it, instead of require the hook directly, you need to declare a `index.js` for load and export all the all the `useStaticQuery`'s:
```

Comment:
You don't actually have to do this. Perhaps remove this?

Reply:
done 🙂